pkgs/by-name/ta/taler-wallet-core/package.nix

```
@@ -0,0 +1,114 @@
+{
+  lib,
+  stdenv,
+  buildGoModule,
+  fetchFromGitHub,
+  fetchgit,
+  git,
+  jq,
+  nodejs,
+  python3,
+  zip,
+  pnpm,
+  removeReferencesTo,
+  srcOnly,
+}:
+let
+  nodePackages = nodejs.pkgs;
+  nodeSources = srcOnly nodejs;
+  esbuild_0_19_9 = buildGoModule rec {
```

You can use
```suggestion
  esbuild_0_19_9 = esbuild.override {
```
instead. See e.g. the deltachat-desktop package.